src/handlers.rs

```diff
@@ -318,6 +324,18 @@ fn handle_read_file_sync(d: *const DenoC, filename: &str) {
   );
 }
 
+// Prototype https://github.com/denoland/deno/blob/golang/os.go#L171-L184
```

Comment:
The link is incorrect - use: https://github.com/denoland/deno/blob/golang/os.go#L186-L195